.gitignore

```diff
@@ -24,6 +24,6 @@ coverage.xml
 
 # Editors
 .vscode/
-
+.idea/
```

Comment:
I'm not sure what this is, but it isn't needed for our project. Could you please remove it from the file?

Reply:
so this was just for Pycharm IDE. So if you use vscode as IDE the settings for the same are added under .vscode. Similarly, for IntelliJ products (Pycharm here), we get .idea to store the IDE settings. This shouldn't be a breaking change, however, I'll still remove it for the timebeing